test/fetch.test.js

```diff
@@ -501,10 +501,12 @@ describe("Fetch AuthHttpRequest class tests", function () {
         });
         try {
             const page = await browser.newPage();
-            await page.goto(BASE_URL + "/index.html", { waitUntil: "load" });
+            // NOTE: Using localhost:8080 as the base URL because browsers ignore
```

Comment:
Please confirm that these tests are duplicated by the test generator and remove them from the "legacy" test files.

Reply:
Yup, these were indeed duplicated. I have removed them.